db.go

```diff
@@ -862,6 +863,10 @@ func (d *DB) makeRoomForWrite(b *Batch) error {
 		// want to go through the flush path in order to recycle that WAL file.
 		imm := d.mu.mem.mutable
 		d.mu.mem.mutable = newMemTable(d.opts)
+		// NB: When the immutable memtable is flushed to disk it will apply a
+		// versionEdit to the manifest telling it that log files <= newLogNumber
```

Comment:
Should it be `log files < newLogNumber`?

Reply:
Huh, well I can read this both ways. I'm going to try and clarify.